### DISCUSSION TITLE
mount: key in values does nothing

### DISCUSSION THREAD

**Discussion by ohthehugemanatee** [ressu/kube-plex#22] [General]
I've mounted my movies with the `extradata` key, and added `mounts: /data,/transcode/data-media` to my helm values. But the transcoder dies on failure to find the file. `kubectl describe` shows that `/data-media` shows it is not mounted, and searching the repo for "mounts" shows it doesn't appear in the codebase anywhere.

**Comment by ressu** [comment]
Hmm.. Is `/data-media` mounted on the plex pod? Also does the main plex pod have an `kube-plex/mounts` annotation? I'm mostly suspecting some typo somewhere but no matter how much I look at the code I can't spot anything, so hopefully we can see the issue on the final deployment 😄
